src/services/services.ts

```diff
@@ -1011,6 +1014,17 @@ module ts {
         isWriteAccess: boolean;
     }
 
+    export interface DocumentHighlights {
+        fileName: string;
+        highlightSpans: HighlightSpan[];
+    }
+
+    export interface HighlightSpan {
+        textSpan: TextSpan;
+        isDefinition: boolean;
+        isWriteAccess: boolean;

```

Comment:
@mhegazy do we still need this? I recall Monaco using it. Can @jrieken weigh in?